libtenzir/src/format/zeek.cpp

```diff
@@ -534,6 +537,25 @@ class zeek_printer : public printer_base<zeek_printer<Iterator>> {
     return true;
   }
 
+  bool operator()(Iterator& out, const view<blob>& x) const {
```

Comment:
So much of the work in this PR is for the old formats; at some point we should really remove them. I've reviewed these parts only very broadly.

Reply:
That's when we effectively break Threat Bus and should also stop offering the `vast` symlink for backwards compatibility. It shouldn't be done as part of this thrust, but I do agree that we should do it soon.